OpenEdXMobile/res/values/colors.xml

```diff
@@ -24,7 +24,7 @@
     <color name="cyan_text_navigation_20">#20299ED7</color>
     <color name="disabled_chapter_list">#EBEBED</color>
     <color name="grey_light">#3c4045</color>
-    <color name="grey_medium">#666666</color>
+    <color name="grey_medium">#707070</color>

```

Comment:
Can you share the reference for this?

Reply:
you can check the Figma design -> Discussion screen -> Read discussion Item.

Reply:
ok